docs/src/components/Footer.tsx

```diff
@@ -61,7 +61,7 @@ const StyledFooterLink = styled(Link)`
 `;
 
 function Dot() {
-  return <Desktop>·</Desktop>;
+  return <Hide on={["smallMobile", "mediumMobile", "largeMobile", "tablet"]}>·</Hide>;
```

Comment:
do we want to also deprecate `Hide` down the road? we should just use TW classes, e.g. `className="hidden dt:block"`

Reply:
nope, Hide is still useful and should be probably deprecated after migration is fully over and users will also use the tailwind.

Reply:
ok let's deprecate `Hide` later